docs/content/en/docs/getting-started/baremetal/bare-prereq.md

```diff
@@ -81,18 +81,4 @@ Here are other network requirements:
 
 ## Validated hardware
 
-Through extensive testing in a variety of on-premises environments, we expect Amazon EKS Anywhere on bare metal to run on most generic hardware that meets the above requirements.
-In addition, we have collaborated with our hardware original equipment manufacturer (OEM) partners to provide you a list of validated hardware:
-
-| Bare metal servers  | BMC   | NIC     |
-|---------------------|-------|---------|
-| Dell PowerEdge R740 | iDRAC9 |  Mellanox ConnectX-4 LX 25GbE  |
-| Dell PowerEdge R7525 (NVIDIA Tesla™ T4 GPU's) | iDRAC9 |  Mellanox ConnectX-4 LX 25GbE & Intel Ethernet 10G 4P X710 OCP |
-| Dell PowerFlex (R640) | iDRAC9 | Mellanox ConnectX-4 LX 25GbE |
-| SuperServer SYS-510P-M | IPMI2.0/Redfish API | Intel® Ethernet Controller i350 2x 1GbE |
-| Dell PowerEdge R240 | iDRAC9 | Broadcom 57414 Dual Port 10/25GbE |
-| HPE ProLiant DL20 | iLO5 | HPE 361i 1G |
-| HPE ProLiant DL160 Gen10 | iLO5 | HPE Eth 10/25Gb 2P 640SFP28 A |
-| Dell PowerEdge R340 | iDRAC9 | Broadcom 57416 Dual Port 10GbE |
-| HPE ProLiant DL360 | iLO5 | HPE Ethernet 1Gb 4-port 331i |
-| Lenovo ThinkSystem SR650 V2 | XClarity Controller Enterprise v7.92 | Intel I350 1GbE RJ45 4-port OCP & Marvell QL41232 10/25GbE SFP28<br>2-Port PCIe Ethernet Adapter |
+Through extensive testing in a variety of on-premises environments, we expect Amazon EKS Anywhere on bare metal to run on most generic hardware that meets the above requirements.  Compatibility is determined by the host operating system selected when [Building Node Images](https://anywhere.eks.amazonaws.com/docs/osmgmt/artifacts/#building-node-images).  Installation may require you to [Customize HookOS for EKS Anywhere on Bare Metal](https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/customize/bare-custom-hookos/) to add drivers, or modify configuration specific to your environment.  [Bottlerocket support for bare metal was deprecated with the EKS Anywhere v0.19 release.](https://github.com/aws/eks-anywhere/issues/7754)
```

Comment:
One more slight tweak, "Through extensive testing in a variety of on-premises environments, we have validated Amazon EKS Anywhere on bare metal works without modification on most modern hardware that meets the above requirements."